src/components/styles.scss

```diff
@@ -0,0 +1,96 @@
+.board {
+  display: flex;
+  justify-content: space-around;
+  min-height: 550px;
+  padding: 10px;
+  border-radius: 8px;
+  min-width: 100%;
+  box-sizing: border-box;
+  background-color: #ccd6c9;
+  .board_title {
+    display: flex;
+    justify-content: center;
+    font-size: 20px;
+    color: #2c3e5d;
```

Comment:
you could use scss variables so that you would be able to change colors and stuff easily

Reply:
fixed